Week2/todo_app/program.js

```diff
@@ -21,39 +21,100 @@ class TodoModel {
         });
     }
 
-    create(description, callback) {
+    create(description, userId, callback) {
         // Write code and query to create a new TODO item
+        const createTodoItem = `INSERT INTO todo_items(text, is_completed, user_id) 
+                                VALUES (${dbConnection.escape(description)}, "0", ${dbConnection.escape(userId)})`;
```

Comment:
We talked about using prepared statement provided my mysql.js to make you write less code in escaping, do check it out :)